js/wasm/src/webworker/index.ts

```diff
@@ -169,6 +169,13 @@ matplotlib.use("agg")
 	await pyodide.runPythonAsync(unloadModulesPySource);
 	unload_local_modules = pyodide.globals.get("unload_local_modules");
 	console.debug("Python utility functions are set up.");
+
+	console.debug("Set environment variables.");
+	await pyodide.runPythonAsync(`
+import os
+os.environ["GRADIO_ANALYTICS_ENABLED"] = "False"
+`);
+	console.debug("Environment variables are set.");
```

Comment:
Not sure, but this might affect their "regular" Python Gradio apps?

Anyways, I don't think we need this anymore

Reply:
This code snippet is an initializer of the Wasm environment, so it doesn't affect the regular usage of the Gradio package.

Reply:
Ah I see in that case, ignore my comments. The PR is good as is